src/common/download.cpp

```diff
@@ -73,8 +72,11 @@ namespace tools
     {
       boost::unique_lock<boost::mutex> lock(control->mutex);
       std::ios_base::openmode mode = std::ios_base::out | std::ios_base::binary;
-      uint64_t existing_size = 0;
-      if (epee::file_io_utils::get_file_size(control->path, existing_size) && existing_size > 0)
+      boost::system::error_code ec{};
+      uint64_t existing_size = static_cast<uint64_t>(boost::filesystem::file_size(boost::filesystem::path(control->path), ec));
```

Comment:
```suggestion
      const uint64_t existing_size = static_cast<uint64_t>(boost::filesystem::file_size(control->path, ec));
```

make `const` and remove explicit conversion to `path`